WMF Framework/Event Platform Client/EventPlatformClientLibrary.swift

```diff
@@ -362,6 +368,12 @@ public class EPC {
         }
     }
 
+    func getStreamConfig() -> [String: [String: Any]]? {
```

Comment:
Can you also reference this method instead of using `streamConfigurations` directly in the `configure()` method and `inSample(stream:)` methods?